docs/site/src/demos/bottom-navigation/LabelBottomNavigation.js

```diff
@@ -32,19 +36,19 @@ export default class LabelBottomNavigation extends Component {
         >
           <BottomNavigationButton
             label="Recents"
-            icon={<span className="material-icons">restore</span>}
+            icon={<RestoreIcon />}
```

Comment:
It doesn't look like removing `display: block` is going to work as a quick fix like it did for Tabs...

This could take me a while - I'm a bit rusty on my CSS!